src/dist/replication/lib/replica.cpp

```diff
@@ -166,6 +170,8 @@ void replica::on_client_read(dsn::message_ex *request)
             response_client_read(request, ERR_INVALID_STATE);
             return;
         }
+    } else {
+        _counter_table_level_backup_request_qps->increment();
```

Comment:
don't (can't) distinguish request types(get, multi_get..)?

Reply:
If we need to distinguish get/multi_get, we should add the perf-counter in pegasus, not rdsn. But in pegasus, we can't get whether the request is backup request or not.
But there is still one way to implement it, just like the way `table level latency` use. But I think it is too complicated.

Reply:
Well, if can implement, we should consider whether distinguish request types and I think it is necessary.

Reply:
Why it is necessary? I think what we need is the total qps of backup request(which means the added server load), but the request type is not very important.